lukasz-lapczyk/review-dealer
# Add user's company

# Overview
Added Company model that belongs to User. Registered user can add new and edit/remove existing company. Furthermore, user can have 1 company created. 

# Testing prerequisites
- bundle install
- db migrations

# Testing
- navigate to root directory
- log in with 'user_1@example.org', pass: Secret19 or create new account
- [x] 'Create company' link should appear on navbar

- click on 'Create company'
- fill in form with company name and click Create
- [ ] company should appear on company list
- [x] 'Your company' followed by company name should appear on navbar

- click on 'Your company' link
- fill in form with new company name and click Update
- [x] updated company data should appear both on listing and navbar

- click on 'Your company' link
- click on 'Delete' button
- [x] company should be deleted
